test/integration/component/test_ps_resize_volume.py

```diff
@@ -48,59 +51,69 @@
                           RESOURCE_SECONDARY_STORAGE,
                           XEN_SERVER)
 
+
 class TestResizeVolume(cloudstackTestCase):
 
     @classmethod
     def setUpClass(cls):
         cloudstackTestClient = super(TestResizeVolume,
-                               cls).getClsTestClient()
+                                     cls).getClsTestClient()
         cls.api_client = cloudstackTestClient.getApiClient()
+        cls.hypervisor = cloudstackTestClient.getHypervisorInfo()
         # Fill services from the external config file
         cls.services = cloudstackTestClient.getParsedTestDataConfig()
         # Get Zone, Domain and templates
         cls.domain = get_domain(cls.api_client)
-        cls.zone = get_zone(cls.api_client, cloudstackTestClient.getZoneForTests())
+        cls.zone = get_zone(
+            cls.api_client,
+            cloudstackTestClient.getZoneForTests())
         cls.services["mode"] = cls.zone.networktype
+        cls._cleanup = []
+        cls.unsupportedStorageType = False
+        if cls.hypervisor.lower() == 'lxc':
+            if not find_storage_pool_type(cls.api_client, storagetype='rbd'):
+                cls.unsupportedStorageType = True
+                return
         cls.resourcetypemapping = {RESOURCE_PRIMARY_STORAGE: 10,
                                    RESOURCE_SECONDARY_STORAGE: 11}
 
-
-
         cls.template = get_template(
-                            cls.api_client,
-                            cls.zone.id,
-                            cls.services["ostype"]
-                            )
+            cls.api_client,
+            cls.zone.id,
+            cls.services["ostype"]
+        )
 
         cls.services["virtual_machine"]["zoneid"] = cls.zone.id
         cls.services["virtual_machine"]["template"] = cls.template.id
         cls.services["volume"]["zoneid"] = cls.zone.id
 
-        cls._cleanup = []
         try:
             cls.hypervisor = str(get_hypervisor_type(cls.api_client)).lower()
 
             # Creating service offering with normal config
             cls.service_offering = ServiceOffering.create(cls.api_client,
-                                            cls.services["service_offering"])
+                                                          cls.services["\
```

Comment:
this looks strange. are you sure you want to break the line within a string-literal?